test/Tests/src/Network/Http_Spec.enso

```diff
@@ -50,156 +51,192 @@ spec =
-            response = HTTP.post url_post "hello world"
+            response = Data.post url_post "hello world"
             response.at "data" . should_equal "hello world"
             response.at "headers" . at "Content-Length" . should_equal "11"
             response.at "headers" . at "Content-Type" . should_equal "text/plain"
 
         Test.specify "Can skip auto-parse" <|
-            response = HTTP.post url_post (Request_Body.Text "hello world") try_auto_parse_response=False
-            response.body.bytes.length . should_equal 383
+            response = Data.post url_post (Request_Body.Text "hello world") try_auto_parse_response=False
+            response.body.bytes.length . should_equal 417
 
         Test.specify "Can send a custom header" <|
-            response = HTTP.post url_post (Request_Body.Text "hello world") headers=[Header.new "Custom" "asdf"]
+            response = Data.post url_post (Request_Body.Text "hello world") headers=[Header.new "Custom" "asdf"]
             response.at "data" . should_equal "hello world"
             response.at "headers" . at "Content-Length" . should_equal "11"
             response.at "headers" . at "Content-Type" . should_equal "text/plain"
             response.at "headers" . at "Custom" . should_equal "asdf"
 
-        Test.specify "Cannot POST when output context is disabled" <|
-            Context.Output.with_disabled <|
-                HTTP.post url_post (Request_Body.Text "hello world") . should_fail_with Forbidden_Operation
-
         Test.specify "Can perform a Request_Body.Text PUT" <|
-            url_put = base_url_with_slash + "put"
-            response = HTTP.post url_put (Request_Body.Text "hello world") method=HTTP_Method.Put
+            response = Data.post url_put (Request_Body.Text "hello world") method=HTTP_Method.Put
             response.at "data" . should_equal "hello world"
             response.at "headers" . at "Content-Length" . should_equal "11"
             response.at "headers" . at "Content-Type" . should_equal "text/plain"
 
         Test.specify "Can perform a Request_Body.Text PATCH" <|
-            url_patch = base_url_with_slash + "patch"
-            response = HTTP.post url_patch (Request_Body.Text "hello world") method=HTTP_Method.Patch headers=[Header.content_type "application/diff"]
+            response = Data.post url_patch (Request_Body.Text "hello world" content_type="application/diff") method=HTTP_Method.Patch
             response.at "data" . should_equal "hello world"
             response.at "headers" . at "Content-Length" . should_equal "11"
             response.at "headers" . at "Content-Type" . should_equal "application/diff"
 
+        Test.specify "Can perform a DELETE" <|
+            response = Data.post url_delete method=HTTP_Method.Delete #try_auto_parse_response=False
+            response.at "headers" . at "Content-Length" . should_equal "0"
+
+        Test.specify "Cannot perform request when output context is disabled" <|
```

Comment:
```suggestion
        Test.specify "Cannot perform POST/PATH/PUT/DELETE request when output context is disabled" <|
```

GET is allowed so the title should not imply that _any_ request is not allowed.

Reply:
Done.